docs/manual-language-docs/lsp-sqls.md

```diff
@@ -2,10 +2,28 @@
 
 
 ``` emacs-lisp
+(add-hook 'sql-mode 'lsp)
```

Comment:
... and it should be `sql-mode-hook`

Reply:
doh! I switched to use-package `:hook ((sql-mode . lsp-deferred))` for myself but didn't want to complicate things … fixed.